src/renderer.cpp

```diff
@@ -631,33 +686,52 @@ void Renderer::draw()
             if (bgfx::isValid(handle))
             {
                 uint32_t flags = 0;
+                
+                if (internalState.bIsMovie) flags |= BGFX_SAMPLER_U_CLAMP | BGFX_SAMPLER_V_CLAMP | BGFX_SAMPLER_W_CLAMP;
 
-                if (backendProgram == RendererProgram::POSTPROCESSING)
-                {
-                    flags = BGFX_SAMPLER_U_CLAMP | BGFX_SAMPLER_V_CLAMP | BGFX_SAMPLER_W_CLAMP | BGFX_SAMPLER_MIN_ANISOTROPIC | BGFX_SAMPLER_MAG_ANISOTROPIC;
-                }
-                else
-                {
-                    if (internalState.bIsMovie) flags |= BGFX_SAMPLER_U_CLAMP | BGFX_SAMPLER_V_CLAMP | BGFX_SAMPLER_W_CLAMP;
-
-                    if (!internalState.bDoTextureFiltering || !internalState.bIsExternalTexture) flags |= BGFX_SAMPLER_MIN_POINT | BGFX_SAMPLER_MAG_POINT | BGFX_SAMPLER_MIP_POINT;
+                if (!internalState.bDoTextureFiltering || !internalState.bIsExternalTexture) flags |= BGFX_SAMPLER_MIN_POINT | BGFX_SAMPLER_MAG_POINT | BGFX_SAMPLER_MIP_POINT;
 
-                    if (flags == 0) flags = UINT32_MAX;
-                }

```

Comment:
What happened to this entire block of code? Why is it gone?

Reply:
Sorry, my mistake, I reverted the changes so it should be as before now.